includes/admin/register-settings.php

```diff
@@ -234,6 +234,13 @@ function wzkb_get_registered_settings() {
 					'type'             => 'text',
 					'options'          => 'kb-tags',
 				),
+				'show_exerpt'     => array(
+					'id'               => 'show_exerpt',
+					'name'             => esc_html__( 'Show exerpt', 'knowledgebase' ),
+					'desc'             => esc_html__( 'Show exerpt under article link', 'knowledgebase' ),
```

Comment:
Description should be "Select to include the post excerpt after the article link"